html-forms/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+  <form action="" method="get">
+    <label for="email">
+      Email
+    </label>
+    <input type="text" name="email" />
```

Comment:
The `for` attribute on `label` elements require a matching `id` attribute on their target `input`. Please add an `id` to each input that has a `label`